actionview/lib/action_view/railtie.rb

```diff
@@ -33,6 +33,8 @@ class Railtie < Rails::Railtie # :nodoc:
         if app.config.action_view.cache_template_loading.nil?
           ActionView::Resolver.caching = app.config.cache_classes
         end
+
+        PartialRenderer.collection_cache = Rails.cache
```

Comment:
The Railties tests fail because `PartialRenderer` doesn't respond to `collection_cache=` here. But I don't understand why it doesn't.

`PartialRenderer` is set to be autoloaded, so this reference should require `CollectionCaching` and include it into the renderer. I'm all ears if anybody has any ideas.